Cubical/Categories/Instances/Preorders/Base.agda

```diff
@@ -14,15 +15,15 @@ private
 
 open Category
 
--- Category of Posets
+-- Category of Preorders
```

Comment:
Do we use this for anything anymore?

Reply:
Not really. Don't see an issue with keeping it just in case, however.